languages.toml

```diff
@@ -2421,3 +2421,17 @@ language-server = { command = "nimlangserver" }
 [[grammar]]
 name = "nim"
 source = { git = "https://github.com/aMOPel/tree-sitter-nim", rev = "240239b232550e431d67de250d1b5856209e7f06" }
+
+[[language]]
+name = "hurl"
```

Comment:
Should we also add `# http` so people can search for it? I wonder.

Since the name hurl is not really useful.

![Screenshot_20230407_002528](https://user-images.githubusercontent.com/4687791/230439500-ac428e19-b8f5-4b4a-889f-1c575738b9b5.png)

I like how doom emacs did it, they put some more description (and funny) description beside the original plugin so people know what is it for.